doc/scapy/troubleshooting.rst

```diff
@@ -50,11 +64,20 @@ With IPv6, you can simply do::
     <IPv6  version=6 tc=0 fl=866674 plen=8 nh=ICMPv6 hlim=64 src=::1 dst=::1 |<ICMPv6EchoReply  type=Echo Reply code=0 cksum=0x7ebb id=0x0 seq=0x0 |>>
 
     # Layer 2
-    >>> conf.iface = "lo"
-    >>> srp1(Ether() / IPv6() / ICMPv6EchoRequest())
+    >>> srp1(Ether() / IPv6() / ICMPv6EchoRequest(), iface=conf.loopback_name)
```

Comment:
What are the resulting IPv6 addresses? Why is the `iface` parameter important after the PR?

Reply:
I'm replacing the `conf.iface = "lo"` (line above), by `iface=` in the sniff itself. Do you think that's confusing?